src/core/Accordion/Accordion.stories.tsx

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import Accordion, { AccordionProps } from "../Accordion";
 
 export default {
-  title: "Components/Accordion",
+  title: "JS Components/Accordion",
```

Comment:
All these changes do is put all these assets in a different "folder" in the Storybook UI - necessary here as I've put the CSS-only stuff in its own "folder".